meson.build

```diff
@@ -0,0 +1,23 @@
+project(
+    'hostpwrctl',
+    'cpp',
+    default_options: [
+        'warning_level=3',
+        'werror=true',
+        'cpp_std=c++17',
```

Comment:
What an ancient! Use 20 :)

Reply:
The Vegman's SDK contains gcc-10.1.0 that says:
```
  -std=c++20                  Conform to the ISO 2020 C++ draft standard
                              (experimental and incomplete support).  Same as
                              -std=c++2a.
  -std=c++2a                  Conform to the ISO 2020 C++ draft standard
                              (experimental and incomplete support).
```
So, I think it's too early.

Reply:
Misspell: 
occured -> occurred
gracefuly -> gracefully
opeation -> operation

Otherwise LGTM

Reply:
misspells fixed